CMakeLists.txt

```diff
@@ -111,12 +111,19 @@ message(STATUS "Targetting ${BUILD_BITS}bit")
 
 ### Install paths (relative to CMAKE_INSTALL_PREFIX)
 if    (UNIX AND NOT MINGW)
-	set(BINDIR  "bin"                CACHE STRING "Where to install binaries")
-	set(LIBDIR  "lib"                CACHE STRING "Where to install libraries")
-	set(MANDIR  "share/man"          CACHE STRING "Where to install man pages")
-	set(DOCDIR  "share/doc/spring-VERSION" CACHE STRING "Where to install basic project documentation (README, LICENSE, etc.)") # TODO set correct version here, and re-enable userdocs in doc/CMakeLists.txt
-	set(DATADIR "share/games/spring" CACHE STRING "Where to install game content")
+	if (INSTALL_PORTABLE)
+		set(BINDIR  "."   CACHE STRING "Where to install binaries")
+		set(LIBDIR  "."   CACHE STRING "Where to install libraries")
+		set(DATADIR "."   CACHE STRING "Where to install game content")
+		set(DOCDIR, "doc" CACHE STRING "Where to install basic project documentation (README, LICENSE, etc.)")
+	else()
+		set(BINDIR  "bin"                CACHE STRING "Where to install binaries")
+		set(LIBDIR  "lib"                CACHE STRING "Where to install libraries")
+		set(DATADIR "share/games/spring" CACHE STRING "Where to install game content")
+		set(DOCDIR  "share/doc/spring-VERSION" CACHE STRING "Where to install basic project documentation (README, LICENSE, etc.)") # TODO set correct version here, and re-enable userdocs in doc/CMakeLists.txt
+	endif()
 
+	set(MANDIR  "share/man"          CACHE STRING "Where to install man pages")
```

Comment:
we have manpages?

Reply:
I don't see any anymore https://github.com/search?q=repo%3Abeyond-all-reason%2Fspring%20MANDIR&type=code, I was just copying code here, this line isn't changed: stuff was moved around it.

Reply:
Looks removed in #1291. But if the line technically wasn't touched it's fine to keep it.

Reply:
I don't want to just drop it as it's set in a bunch of places